docs/source/UserGuide/VUDRM-Admin.md

```diff
@@ -1,44 +1,44 @@
 # USER GUIDE
 
-Our VUDRM Admin page gives you the tools you will need to test our DRM services with your content. If you need assistance or clarification in relation to anything in this guide, please contact <support@vualto.com>.
+Our Studio DRM Admin page gives you the tools you will need to test our DRM services with your content. If you need assistance or clarification in relation to anything in this guide, please contact <support@vualto.com>.
```

Comment:
change support email to support@jwplayer.com

Reply:
Have updated.